packages/lit-element/package.json

```diff
@@ -93,7 +93,7 @@
     "tslib": "^2.0.3",
     "typescript": "^4.1.3"
   },
-  "typings": "lit-element.d.ts",
+  "typings": "index.d.ts",
```

Comment:
Do we even need this field anymore? I feel like TypeScript looks here by default now.

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html says

"Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so."

Reply:
It does seems to work also if this is deleted

Reply:
Let's keep it, but it does like like `"types"` is the newer field.

Reply:
Changed to 'types' and moved it next to other index files